hack/verify-import-aliases.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${SCRIPT_ROOT}"
+
+ROOT_PATH=$(pwd)
+GOPATH=$(go env GOPATH | awk -F ':' '{print $1}')
+IMPORT_ALIASES="-import-aliases $ROOT_PATH/hack/.import-aliases"
+INCLUDE_PATH="-include-path (cmd|pkg|test/e2e|test/helper)"
+CONFIRM="-confirm"
+
+ret=0
+read -p  "Do you want to update file automatically [y/n] " choice
+case ${choice} in
```

Comment:
I don't think we need to `update` the alias in a `verification` script. They should be in `update-import-aliases.sh`.